src/features/images/imagesSlice.js

```diff
@@ -240,6 +240,22 @@ export const imagesSlice = createSlice({
       const index = payload;
       state.loadingStates.export.errors.splice(index, 1);
     },
+
+    deleteImagesStart: (state) => {
+      state.loadingStates.images.isLoading = true;
+      state.loadingStates.images.operation = 'deleting';
+      state.loadingStates.images.error = null;
+    },
+
+    deleteImagesSuccess: (state, { payload }) => {
+      state.loadingStates.images.isLoading = false;
+      state.loadingStates.images.operation = null;
+    },
+
+    deleteImagesError: (state, { payload }) => {
```

Comment:
Like create-projects and manage-users, we'll also want to wire up `ErrorAlerts` for any errors returned by the API.

Reply:
This just didn't work because of a couple of types, which are now fixed.